core/src/main/resources/hudson/triggers/Messages_de.properties

```diff
@@ -25,5 +25,8 @@ SCMTrigger.getDisplayName={0} Abfrage-Protokoll
 SCMTrigger.BuildAction.DisplayName=Abfrage-Protokoll
 SCMTrigger.SCMTriggerCause.ShortDescription=Build wurde durch eine SCM-�nderung ausgel�st.
 TimerTrigger.DisplayName=Builds zeitgesteuert starten
+TimerTrigger.MissingWhitespace=Es scheinen Leerzeichen zwischen * und * zu fehlen.
+TimerTrigger.no_schedules_so_will_never_run=Keine zeitgesteuerten Builds, somit wird Job nie laufen
```

Comment:
Not sure this unambiguously explains the issue. It's missing *schedules* (i.e. cron expressions), therefore this will never trigger the build. Since I'm not using Jenkins in German I'm also not sure *laufen* is the correct term.